transmogrifier/sources/springshare.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from typing import List, Optional
+
+from bs4 import Tag
+from dateutil.parser import parse as date_parser
+
+import transmogrifier.models as timdex
+from transmogrifier.helpers import validate_date
+from transmogrifier.sources.oaidc import OaiDc
+
+logger = logging.getLogger(__name__)
+
+
+class SpringshareOaiDc(OaiDc):
+    """
+    Springshare transformer that extends generic OAI DC transformer.
+
+    This transformer is used for:
+        - libguides
+        - researchdatabases
+    """
+
+    def get_dates(self, xml: Tag) -> Optional[List[timdex.Date]]:
```

Comment:
I would add unit tests for both `springshare.get_dates` and `springshare.get_links` covering the possible data scenarios (success, unparseable date, etc.) to make sure nothing can crash the transform